packages/storybook8/stories/Concepts/TogetherMode/Docs.mdx

```diff
@@ -0,0 +1,62 @@
+import { Meta, Source } from '@storybook/addon-docs';
+
+<Meta title="Concepts/TogetherMode" />
+
+import PPTLiveSnippetText from '!!raw-loader!./snippets/TogetherMode.snippet.tsx';
+
+# TogetherMode
+
+Together Mode in Teams uses AI to place everyone in a shared virtual background, creating the feeling of being in the same room.
+It’s built to make meetings more engaging and foster stronger connections.
+
+It’s especially effective for training sessions, workshops, or team-building activities.
+A classroom setting, like the one shown below, promotes learning and collaboration, helping participants stay engaged and comfortable in a familiar atmosphere.
+
+<img style={{ width: 'auto', height: 'auto' }} src="images/together-mode/togethermode-scenario_example.png" />
+
+## Supported Scenario
+
+Together Mode is available only in Teams meetings and group calls. It can be started exclusively by Teams users with the roles of Organizer, Co-organizer, or Presenter.
+ACS (Azure Communication Services) users cannot initiate Together Mode themselves, but they can view the Together Mode stream if it’s started by a Teams user.
+
+<sub>
+  <sup>Layout Menu Displaying the Together Mode Option as Disabled When a Teams User Hasn't Started Together Mode</sup>
```

Comment:
```suggestion
  <sup>Layout menu displaying the Together Mode option as disabled When a Teams user hasn't started Together Mode</sup>
```

Reply:
Not entirely convinced we need a <sup>

Reply:
As the sup looks pretty odd here
![image](https://github.com/user-attachments/assets/ff316daf-7fdf-4389-9d0b-226c165a4481)